source/deployment/mobile-app-deployment.rst

```diff
@@ -413,16 +393,16 @@ Consult the `Mattermost mobile app changelog <https://github.com/mattermost/matt
 ----
 
 ###################################
-Mobile Deployment via Custom Builds
+Mobile deployment via custom builds
 ###################################
 
-Choosing this model means you’ve decided not to use the mobile applications Mattermost has made available via the public app stores. 
+Choosing this model means you have decided not to use the mobile applications Mattermost has made available via the public app stores. 
 
 This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost server.
```

Comment:
```suggestion
This also means you will need to maintain these applications. Maintenance includes rebuilding and incorporating feature and/or security updates. Otherwise, your applications will not match the functionality of our publicly available applications, and could be incompatible with future versions of the Mattermost Server.
```